README.md

```diff
@@ -1 +1,90 @@
 # firewatch
+
+The firewatch project is in its infancy and currently has limited functionality. Ultimately, we'd like this tool to be a way to manage, track, and react to job failures in OpenShift CI.
+
+**What may be added in future?**
+
+- Automatic re-tries for user-defined failures in an OpenShift CI job.
+
+## Features
+
+- Automatically creates Jira issues for failed OpenShift CI jobs.
+  - The issues created are provided with any logs or junit files found in a failed step.
+  - The issues can be created in any Jira instance or project (provided the proper credentials are supplied).
+  - If multiple issues are created for a run, the issues will all be related to each other to help with issue.
+  - The issues created have a "classification" section that is defined by the user in their firewatch config.
+    - This can be used as a "best-guess" for what may have gone wrong during the job's run.
+
+## Getting Started
+
+### Usage in OpenShift CI
+
+Reporting issues using this tool in OpenShift CI is very simple, you can do one of the following:
+
+- Add the [`firewatch-report-issues` ref](https://github.com/openshift/release/tree/master/ci-operator/step-registry/firewatch/report-issues) to the `post` steps of a test. This job should be last, if that is not possible, make sure it is at least following any step that you'd like an issue generated for.
+- Use a workflow with the [`firewatch-report-issues` ref](https://github.com/openshift/release/tree/master/ci-operator/step-registry/firewatch/report-issues) already in the `post` steps. For example, see the [`firewatch-ipi-aws` workflow](https://github.com/openshift/release/tree/master/ci-operator/step-registry/firewatch/ipi/aws).
+
+Remember, when you are using the `firewatch-report-isssues` ref, some variables need to be defined in your configuration file:
+
+- `FIREWATCH_CONFIG` [REQUIRED]
+  - This value should be a list of rules you have defined for firewatch to report on. For more information how to define these rules, please see the [CLI usage guide](docs/cli_usage_guide.md#defining-the-configuration).
+  - Example:
+
+    ```yaml
+    FIREWATCH_CONFIG: |
+        [
+            {"step": "exact-step-name", "failure_type": "pod_failure", "classification": "Infrastructure", "jira_project": "PROJECT"},
+            {"step": "*partial-name*", "failure_type": "all", "classification":  "Misc.", "jira_project": "OTHER"},
+            {"step": "*ends-with-this", "failure_type": "test_failure", "classification": "Test failures", "jira_project": "TEST"}
+            {"step": "*ignore*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", ignore: "true"}
+        ]
+    ```
+
+- `FIREWATCH_JIRA_SERVER`
+  - The Jira server to report to.
+  - DEFAULT: `https://issues.stage.redhat.com`
+- `FIREWATCH_JIRA_API_TOKEN_PATH`
```

Comment:
How do we find this token? And for running it in OpenShift CI, where do we provide it

Reply:
Currently, our token is in Vault, I may need to go in and give perms to the rest of the team to see it. As for usage in OpenShift CI, the path is already provided as a default and is mapped to the Vault secret.